haproxy.cfg

```diff
@@ -56,7 +56,7 @@ frontend router
   # Rewrite some urls to our own
   use_backend rewrite-publish if { path '/publish' } || { path_beg /settings/ }
   # Route all these urls to the Elk interface
-  use_backend elk if { path / } || { path '/compose' } || { path /settings } || { path /bookmarks } || { path '/conversations' } || { path '/favourites' } || { path '/home' } || { path '/public' } || { path '/search' }
+  use_backend elk if { path / } || { path '/compose' } || { path /settings } || { path /bookmarks } || { path '/conversations' } || { path '/favourites' } || { path '/home' } || { path '/public' } || { path '/search' } || { path '/discover' }
```

Comment:
you might need to move this to the next line, or indeed, start a line `use_backend elk` line if haproxy doesn't like long lines (I didn't count how many characters there are currently, though)

Reply:
I'll go ahead and move it to the next line just in case